lib/magma/validation_object.rb

```diff
@@ -24,8 +24,8 @@ def options
     def match
     end
 
-    def as_json
-      @options.to_json
+    def to_json
+      @options
```

Comment:
@graft Calling `.to_json` here would convert it to a JSON string before having it converted again by the retrieve endpoint. I left it as a hash to prevent needing to double parse the validation_object#options.

Reply:
The expected behavior of to_json is to return a JSON string. So you should have `@options.to_json` here and not call to_json in `#json_template`. In the test you might call `JSON.parse(attribute.json_template.to_json)` to test the output.